package/README.md

```diff
@@ -0,0 +1,79 @@
+# Release tools
+
+## Overview
+
+This directory contains scripts used to automate the release process of esp8266/Arduino project.
+The purpose of the release process is to generate the following outputs from the repository:
+
+* Boards manager package index for Arduino IDE (i.e. `package_esp8266_index.json`). See [specification](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.6.x-package_index.json-format-specification) of package index file for more info.
+
+* Boards manager package for Arduino IDE. This is a .zip file which contains source code of esp8266/Arduino project, platform.txt, boards.txt and a few other supporting files. See [3-rd party hardware specification](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5-3rd-party-Hardware-specification) for more info about the layout of the boards manager package.
+
+* Github Release for esp8266/Arduino project. This is used to host the boards manager package mentioned above, and also contains the release notes.
+
+Here is an overview of the release process.
+
+1. Release process starts when a maintainer pushes a tag into the repository.
+
+2. Travis CI runs a build for this tag, and one of the jobs (with `BUILD_TYPE=package`) is used to prepare the boards manager package. This job runs `build_boards_manager_package.sh`.
+
+3. `build_boards_manager_package.sh` does a few things to build the boards manager package (.zip) file and the json index:
+
+   *  Pack source files into a zip file, excluding some files and tweaking platform.txt.
+   *  Download current latest package index json file from Github Releases. This file contains descriptions of all previous releases.
+   *  Generate package index for the new release.
+   *  Combines new release with previous releases in one json file (using `merge_packages.py` script).
+
+4. Travis CI uploads boards manager package (.zip file) and package index (.json file) to Github Releases, creating a draft release at the same time.
+
+5. Travis CI also uploads package index .json file to `http://arduino.esp8266.com/stable/package_esp8266_index.json`, i.e. well-known URL used by most users. 
+
+6. When the draft release is created, maintainer edits release description and inserts changelog into the description field, unmarks the release as draft, and publishes the release.
```

Comment:
I think I understand everything except this.  How/where is this draft release created?